cluster/examples/kubernetes/ceph/csi/example/cephfs/storageclass.yaml

```diff
@@ -4,17 +4,13 @@ metadata:
   name: csi-cephfs
 provisioner: cephfs.csi.ceph.com
 parameters:
-  # Comma separated list of Ceph monitors
-  # if using FQDN, make sure csi plugin's dns policy is appropriate.  
-  monitors: mon1:port,mon2:port,...
+  # Specify a string that identifies your cluster. Ceph CSI supports any
+  # unique string. When Ceph CSI is deployed by Rook use the Rook namespace,
```

Comment:
Can we just say this must be the namespace rather than tell them everything csi supports?

Reply:
done